src/entities/constants.ts

```diff
@@ -0,0 +1,21 @@
+import type { RootState } from '../app/store'
+import { Todo, TodosState } from './globalTypes'
+
+export const todosFunctionForStateSelector = (state: RootState): TodosState =>
+  state.todos
+
+export const todosFunctionForTodosSelector = (state: RootState): Todo[] =>
+  state.todos.todos
+
+export const todosFunctionForLoadingSelector = (state: RootState): boolean =>
+  state.todos.loading
```

Comment:
What do you think of these names? Please avoid using the word "function" in the functions names. Also, remember to use `createSelector`.

```suggestion
export const selectTodosReducer = (state: RootState): TodosState =>
  state.todos

export const selectTodos = (state: RootState): Todo[] => createSelector(selectTodosReducer, (state: TodosState) => state.todos)

export const selectTodosLoading = (state: RootState): boolean =>
  createSelector(selectTodosReducer, (state: TodosState) => state.loading)
```

Reference:
* https://redux-toolkit.js.org/api/createSelector
* https://redux.js.org/usage/deriving-data-selectors